nikola/data/themes/base-jinja/templates/base_header.tmpl

```diff
@@ -36,15 +36,15 @@
             <ul>
             {% for suburl, text in url %}
                 {% if rel_link(permalink, suburl) == "#" %}
-                    <li class="active"><a href="{{ permalink }}">{{ text }} <span class="sr-only">{{ messages("(active)", lang) }}</span></a></li>
+                    <li class="active"><a href="{{ permalink }}" aria-label="{{ messages('(active)', lang) }}">{{ text }}</a></li>
```

Comment:
This won’t work as expected.

This HTML snippet:

```html
    Menu:
    <a href="https://getbootstrap.com/">SR <span class="sr-only">(active)</span></a> or
    <a href="https://w3.org/" aria-label="(active)">ARIA</a>.
```

is read out by Safari with VoiceOver like this:

> Menu: *link*, SR (active) or *link*, ARIA.

And when you use Tab to highlight the link in question,

> SR (active), *link*  
> (active), *link*

To make it work in the first case, you **need** `sr-only` or equivalent. To fix the second case, you need to duplicate link text inside of `aria-label`.